deepfry/deepfry.py

```diff
@@ -197,7 +197,7 @@ async def _get_image(self, ctx, link: Union[discord.Member, str]):
 			if discord.version_info.major == 1:
 				avatar = link.avatar_url_as(static_format="png")
 			else:
-				avatar = link.display_avatar.with_static_format("png").url
+				avatar = link.display_avatar
```

Comment:
```suggestion
				avatar = link.display_avatar.with_static_format("png")
```
The `.url` changing it from `Asset` to `str` was the issue, not converting it to `.png`.

Reply:
But do you even need that? It works perfectly without the conversion.

Reply:
There are some subtle differences relating to transparency that are solved by just always pulling the png (or technically webp) version of the image.

Reply:
Good to know, thanks for the explanation.